index.html

```diff
@@ -219,53 +217,53 @@ <h3>Get Digital Permanent Resident Card</h3>
-            Louis's Browser - Holder App->>uscis.gov - Issuer App:uscis.gov/getPRC    
-            uscis.gov - Issuer App->>Louis's Browser - Holder App:request authentication   
-            Louis's Browser - Holder App->>Louis - Holder:request authentication    
-            Louis - Holder->>Louis's Browser - Holder App:provide authentication details   
-            Louis's Browser - Holder App->>uscis.gov - Issuer App:uscis.gov/authenticate?louiss-details   
-            uscis.gov - Issuer App->>uscis.gov - Issuer App:process provided authentication details   
-            uscis.gov - Issuer App->>Louis's Browser - Holder App:authentication succus    
-            Louis's Browser - Holder App->>Louis - Holder:authentication succuss  
-            uscis.gov - Issuer App->>Louis's Browser - Holder App:Holder-App/requestPresentation
-            Louis's Browser - Holder App->>Louis - Holder:Holder/selectHolderService 
-            Louis - Holder->>Louis's Browser - Holder App:holderServiceSelection
-            Louis's Browser - Holder App->>authn.io - Holder Service:HolderService/requestPresentation   
-            authn.io - Holder Service->>Louis - Holder:Holder/selectWallet    
-            Louis - Holder->>authn.io - Holder Service:walletSelection    
-            authn.io - Holder Service->>Louis - Holder:Holder/selectProfile  
-            Louis - Holder->>authn.io - Holder Service:profileSelection   
-            authn.io - Holder Service->>Louis's Browser - Holder App:HolderApp/requestedPresentation   
-            Louis's Browser - Holder App->>uscis.gov - Issuer App:IssuerApp/requestedPresentation   
-            uscis.gov - Issuer App->>Generic Issuer SAAS - Issuer Service:IssuerService/credentials/issue?\"providedVP\"   
-            Generic Issuer SAAS - Issuer Service->>uscis.gov - Issuer App:IssuerApp/credentials/issued?\"issuedCredentailsWrappedByIssuerApp\"    
-            uscis.gov - Issuer App->>Louis's Browser - Holder App:HolderApp/newCredential?\"issuedCredentialUnwrappedByIssuerApp\"    
-            Louis's Browser - Holder App->>Louis - Holder:Holder/selectHolderService 
-            Louis - Holder->>Louis's Browser - Holder App:holderServiceSelection   
-            Louis's Browser - Holder App->>authn.io - Holder Service:HolderService/storeCredentials?\"credentials\"    
-            authn.io - Holder Service->>Louis - Holder:Holder/selectWallet 
-            Louis - Holder->>authn.io - Holder Service:walletSelection   
-            authn.io - Holder Service->>Louis - Holder:Holder/selectProfile  
-            Louis - Holder->>authn.io - Holder Service:profileSelection   
-            authn.io - Holder Service->>authn.io - Holder Service:Store credentials 
-            authn.io - Holder Service->>Louis's Browser - Holder App:HolderApp/credentialsStored
+            sequenceDiagram
+              autonumber
+            Lana (Issuer Admin)->>uscis.gov (Issuer App):Configure to properly use Generic Issuer SAAS
```

Comment:
Does Mermaid allow inline linebreaks (`<br>` or `<br />`)? If so, I strongly suggest putting a linebreak before each parenthetical in these labels, e.g. ---
```suggestion
            Lana<br>(Issuer Admin)->>uscis.gov<br>(Issuer App):Configure to properly use Generic Issuer SAAS
```
(Space chars around the `<br>`s would help readability of these Mermaid source blocks, if it can also handle these.)
```suggestion
            Lana <br> (Issuer Admin)->>uscis.gov <br> (Issuer App):Configure to properly use Generic Issuer SAAS
```
I'm happy to do these PRs, mind numbing as they would be, because I think they'll increase diagram clarity (and possibly allow the tools to increase font sizes, due to less width requirement for the text, if I understand how Mermaid does its job).

Note -- Neither of the suggestions in this comment can be accepted via GitHub, because it can only handle one suggestion per comment. That's fine, as they are insufficient on their own, only presented here as visual aids, and will be trivial to roll into the larger changes I'm really suggesting, if executing this suggestion is viable.

Reply:
The sample "Sequence diagram" a bit below [Diagram Types](https://mermaid-js.github.io/mermaid/#/?id=diagram-types) includes —
```
    Note right of John: Rational thoughts <br/>prevail!
```
— so it does support the clarity-improving `<br/>`s I suggested. :-)

Reply:
I think lets wait until I get my script updates that should handle the <br/> issue. Since we are hoping to do more of this kind of work in the future I would like to get a non-manual way of getting these formatting issues resolved, because as you said it would be very tedious to do manually for all the diagrams.

Reply:
To give a quick update on this. I have the script that I am using to export the mermaid updated to include the <br> changes. Joe and I are holding off on accepting these PRs to wait for some work unifying the URL structure across the documents.